.devcontainer/devcontainer.json

```diff
@@ -14,7 +14,7 @@
   },
 
   "extensions": [
-    "matklad.rust-analyzer",
+    "rust-lang.rust-analyzer",
```

Comment:
Unrelated, but I noticed that rust-analyzer isn't installed in the devcontainer, so I updated the extension id (see https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer).

Reply:
Since it's unrelated to this PR I suggest to open a new PR that will change it

Reply:
It's just one line though? Doesn't seem to worth the hassle.

Reply:
If someone complains that it broke their setup we will revert a commit including changes to "upgrade" subcommand. So yes, it's worth the hassle

Reply:
Addressed in https://github.com/denoland/deno/pull/15751